src/main/java/im/status/keycard/KeycardApplet.java

```diff
@@ -236,10 +267,9 @@ public void process(APDU apdu) throws ISOException {
     }
 
     apdu.setIncomingAndReceive();
-    byte[] apduBuffer = apdu.getBuffer();
 
     try {
-      switch (apduBuffer[ISO7816.OFFSET_INS]) {
```

Comment:
add the INS_LOAD_CERTS and the INS_EXPORT_CERTS flags to this switch block.

Reply:
Resolving because this is a derivative of the other comment